rollup.config.js

```diff
@@ -1,15 +1,21 @@
-import babel from "rollup-plugin-babel";
-import resolve from "rollup-plugin-node-resolve";
-import replace from "rollup-plugin-replace";
-import commonjs from "rollup-plugin-commonjs";
-import filesize from "rollup-plugin-filesize";
-import { uglify } from "rollup-plugin-uglify";
-import ignore from "rollup-plugin-ignore";
-import pkg from "./package.json";
+const { existsSync } = require("fs");
+const { join } = require("path");
+const babel = require("rollup-plugin-babel");
+const resolve = require("rollup-plugin-node-resolve");
+const replace = require("rollup-plugin-replace");
+const commonjs = require("rollup-plugin-commonjs");
+const filesize = require("rollup-plugin-filesize");
+const { uglify } = require("rollup-plugin-uglify");
+const ignore = require("rollup-plugin-ignore");
+const pkg = require("./package.json");
+const publicFiles = require("./scripts/publicFiles");
 
-const { name } = pkg;
-const external = Object.keys(pkg.peerDependencies).concat("prop-types");
-const allExternal = external.concat(Object.keys(pkg.dependencies));
+const external = [...Object.keys(pkg.peerDependencies), "prop-types"];
+const allExternal = [...external, ...Object.keys(pkg.dependencies)];
+
+const multipleInput = Object.values(publicFiles).map(file =>
+  file.replace(/\/index.js$/, "")
+);
```

Comment:
Because `rollup-plugin-commonjs` still doesn't support `input` as objects (https://github.com/rollup/rollup-plugin-commonjs/pull/329), we have to remove `/index.js` from paths so they'll be correctly named with their dirname.